tests/test_edgeql_delete.py

```diff
@@ -494,3 +498,50 @@ async def test_edgeql_update_in_conditional_bad_02(self):
                         (DELETE DeleteTest)
                     );
             ''')
+
+    async def test_edgeql_delete_abstract_01(self):
+        await self.con.execute(r"""
+            SET MODULE test;
+
+            INSERT DeleteTest { name := 'child of abstract 1' };
+            INSERT DeleteTest2 { name := 'child of abstract 2' };
+        """)
+
+        await self.assert_query_result(
+            r"""
+                WITH
+                    MODULE test,
+                    D := (
+                        DELETE
```

Comment:
Do we have tests for UPDATEs on abstract types?

Reply:
We do.  The abstract bit is irrelevant, this was broken for non-abstract inheritance too.